README.md

```diff
@@ -24,107 +24,135 @@ pip install kishu jupyterlab_kishu
+      <img width="350" src="docs/images/init_dropdown.png"/>
+    </td>
+    <td style="background-color: grey; text-align: center;">
+      <img width="350" src="docs/images/init_command.png"/>
+    </td>
+  </tr>
+</table>
+</div>
 
 You will see the below notification if Kishu is initialized successfully:
 
-<br> <!-- HTML tag to force a line break if you want more space -->
-
-<img width="400" src="docs/images/init_success.png"/>
-
-<br> <!-- HTML tag to force a line break if you want more space -->
+<div align="center">
+<img width="350" src="docs/images/init_success.png"/>
+</div>
 
 ### Undoable Notebook Workflow
 **Automatic Tracking**: Once initialized, Kishu will start automatically saving the variable state after each cell execution.
-**Undoing a cell execution:** To undo your latest cell execution, you can use the `Kishu > Initialize/Re-attach` option under the `Kishu` tab (left) or in the Jupyter command palette (right). Alternatively, you can use the shortcut `Ctrl+K then Ctrl+Z` / `⌘+K then ⌘+Z`.
-
-<br> <!-- HTML tag to force a line break if you want more space -->
-
-<img width="350" src="docs/images/undo_dropdown.png"/><img width="350" src="docs/images/undo_command.png"/>
-
-<br> <!-- HTML tag to force a line break if you want more space -->
+**Undoing a cell execution:** To undo your latest cell execution, you can use the `Kishu > Initialize/Re-attach` option under the `Kishu` tab or in the Jupyter command palette. Alternatively, you can use the shortcut `Ctrl+K then Ctrl+Z` / `⌘+K then ⌘+Z`.
+
+<div align="center">
+<table>
+  <tr>
+    <th text-align: center;">`Kishu` Tab</th>
+    <th text-align: center;">Jupyter Command Palette</th>
+  </tr>
+  <tr>
+    <td style="background-color: grey; text-align: center;">
+      <img width="350" src="docs/images/undo_dropdown.png"/>
+    </td>
+    <td style="background-color: grey; text-align: center;">
+      <img width="350" src="docs/images/undo_command.png"/>
+    </td>
+  </tr>
+</table>
+</div>
 
 If successful, you will see the following notification, and your variable state will be rolled back to that before your latest cell execution. 
 
-<br> <!-- HTML tag to force a line break if you want more space -->
-
-<img width="400" src="docs/images/undo_success.png"/>
-
-<br> <!-- HTML tag to force a line break if you want more space -->
+<div align="center">
+  <img width="350" src="docs/images/undo_success.png"/>
+</div>
 
 **Only affects variable state:** Undoing cell executions **only affects the variable state** (right). The code state (i.e., the cells you write, left) is untouched. This can be useful, for example, to 'un-drop' a dataframe column dropped by a cell while keeping the cell code itself intact:
 
-<br> <!-- HTML tag to force a line break if you want more space -->
-
-<img width="600" src="docs/images/workflow_undo.png"/>
-
-<br> <!-- HTML tag to force a line break if you want more space -->
+<div align="center">
```

Comment:
Also crop the white space on the right side of these images, so they're centered properly